npm-shrinkwrap.json

```diff
@@ -1,322 +0,0 @@
-{
```

Comment:
Can you explain your logic in removing the `npm-shrinkwrap.json`? Why isn't it needed?